### PR TITLE
feat(components/ag-grid): add inline help support

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2276340](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2276340)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/595/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/popovers-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/595/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/595/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/595/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/595/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/595/playground/)

File: libs/components/ag-grid/src/lib/modules/ag-grid/header/header-group.component.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <sky-icon *skyThemeIf="'default'" icon="angle-double-right"></sky-icon>
+    <sky-icon
+      *skyThemeIf="'modern'"
+      icon="double-chevron-right"
+      iconType="skyux"
+    ></sky-icon>
+  </button>
+</ng-template>
+<span *ngIf="componentPortal" class="sky-control-help-container">
+  <ng-container [cdkPortalOutlet]="componentPortal"></ng-container>
```
Comment (Blackbaud-SteveBrush) [inline libs/components/ag-grid/src/lib/modules/ag-grid/header/header-group.component.html:38]: Neat.

File: libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts [hunk 2]
```diff
@@ -183,7 +185,7 @@ export class SkyAgGridService implements OnDestroy {
     defaultGridOptions: GridOptions,
     providedGridOptions: GridOptions
   ): GridOptions {
-    const mergedGridOptions = {
+    const mergedGridOptions: GridOptions = {
```
Comment (Blackbaud-SteveBrush) [inline libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts:188]: Good change.

File: libs/components/ag-grid/src/lib/modules/ag-grid/header/header.component.ts [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  public refresh(params: SkyAgGridHeaderParams): boolean {
+    this.agInit(params);
+    return false;
+  }
+
+  #updateSort() {
+    this.sortOrder$.next(this.params.column.getSort() || undefined);
+  }
+
+  #updateSortIndex() {
```
Comment (Blackbaud-SteveBrush) [inline libs/components/ag-grid/src/lib/modules/ag-grid/header/header.component.ts]: ☝️ All of these methods need return types.

**Comment by johnhwhite** [comment]
Moving to Sky UX 7.

**Pull request closed without merging**